rpc-client-api/src/config.rs

```diff
@@ -49,8 +49,14 @@ pub struct RpcSimulateTransactionConfig {
 #[derive(Serialize, Deserialize, Clone, Copy, Debug, Eq, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub enum SimulationSlotConfig {
+    /// Simulate on top of bank with the provided commitment.
```

Comment:
does this simulate on top of current bank? maybe tweak comment to clarify

Reply:
No it fetches the bank with the provided commitment level